crates/rpc/rpc/src/eth/api/pending_block.rs

```diff
@@ -143,20 +215,62 @@ impl PendingBlockEnv {
             gas_limit: block_gas_limit,
             difficulty: U256::ZERO,
             gas_used: cumulative_gas_used,
-            blob_gas_used: None,
-            excess_blob_gas: None,
+            blob_gas_used: Some(sum_blob_gas_used),
```

Comment:
for blob gas used we need to check if cancun is activated via chainspec and only set it if it's activated